src/app/modules/courses/components/edit-course/edit-course.component.html

```diff
@@ -1,24 +1,25 @@
+<app-breadcrumb></app-breadcrumb>
 <form class="form">
```

Comment:
for first time remove form, it wil be in next module, use simple <input>

Reply:
done